js/views/homeVw.js

```diff
@@ -147,7 +147,7 @@ module.exports = Backbone.View.extend({
     item.userID = item.guid;
     var newItemModel = new itemShortModel(item);
     var itemShort = new itemShortView({model: newItemModel});
-    this.$el.find('.js-list1').append(itemShort.el);
+    this.$el.find('.js-list1 .js-loadingMsg').before(itemShort.el);
```

Comment:
Just noting that this will cause new items to appear at the top left, and push the older items down the page as they appear.